api/src/main/java/com/pip/chatbot/controller/CitiesController.java

```diff
@@ -1,86 +1,46 @@
 package com.pip.chatbot.controller;
 
 import com.pip.chatbot.model.City;
-import com.pip.chatbot.repository.CitiesRepository;
+import com.pip.chatbot.payload.response.Response;
+import com.pip.chatbot.payload.response.ResponseStatus;
+import com.pip.chatbot.service.CityService;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
 
 @RestController
 public class CitiesController {
-    private final CitiesRepository citiesRepository;
+    private final CityService cityService;
```

Comment:
Jak już mamy wszędzie liczbę mnogą to niech i serwis będzie Cities:) Konstruktor publiczny, tak jak mamy wszędzie, dla spójności